packages/effect/dtslint/Schema/Brand.tst.ts

```diff
@@ -10,12 +10,9 @@ const schema2 = pipe(Schema.Number, Schema.int(), Schema.brand(Int2))
 type A1 = Schema.Schema.Type<typeof schema1>
 type A2 = Schema.Schema.Type<typeof schema2>
 
-declare const a2: A2
-declare const f: (int: A1) => void
-
 describe("SchemaBrand", () => {
-  it("We should only have one error for the missing definition", () => {
-    // @ts-expect-error
-    f(a2)
+  it("should differentiate between branded schema types", () => {
+    expect<A1>().type.not.toBeAssignableTo<A2>()
+    expect<A2>().type.not.toBeAssignableTo<A1>()
```

Comment:
Looks like this was the intention, or?

Reply:
honestly I don't remember, but your test looks good